### PR TITLE
Prevents pseudo-exploit with heal_on_apply reagents and IV bags

### PR DESCRIPTION
**Pull request by chuga-git**
## What Does This PR Do
This targets a very specific chem mix that has seen use in recent weeks as precisely as possible so that the remaining majority of use cases aren't affected. That is, this mix can no longer heal faster than almost anything in the game, which granted the user near god-mode in combat.
image

As this mix demonstrates, heal_on_apply reagents (styptic powder, silver sulfadiazine, and synthflesh) can be fed via a 1u transfer rate IV drip together at volumes as low as 0.1u per cycle to provide constant, passive, patch-equivalent brute and burn healing. This is just an unfortunate fact of the reagent system. Additionally, the toxin damage that is done on ingestion is proportional to half of the volume. In this specific mix's case, each reagent's volume is 0.1u... so the toxin damage received as a supposed downside is 0.1 per cycle. This is easily healed by the 1 toxin damage per cycle rate of triple citrus.

Instead, I have added a cut off for ingestion healing of the three horsemen below 1u. This means means that there is no 1u-fractional mix that can duplicate the described effect.

## Why It's Good For The Game
Being able to use heal_on_apply reagents in this manner is a serious balance issue. It is so bad that it is considered to be an exploit by Balance Team consensus.

## Testing
I injected myself with a 1u drip and didn't get healed by the relevant reagents. Then I stuck a patch on and got healed.

<hr>

### Declaration

- [x] I confirm that I either do not require [pre-approval](https://github.com/ParadiseSS13/Paradise/blob/master/CODE_OF_CONDUCT.md#types-of-changes-that-need-approval) for this PR, or I have obtained such approval and have included a screenshot to demonstrate this below.
  
  
  <hr>

## Changelog

:cl:
tweak: Styptic powder, silver sulfadiazine, and synthflesh no longer heal when ingested in quantities below 1u.
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/medicine.dm [hunk 1]
```diff
@@ -322,6 +322,9 @@
 	if(method == REAGENT_TOUCH)
 		heal_overall_damage(M, volume)
 
+	if(method == REAGENT_INGEST)
+		data = "Ingested" // sin
```
Comment (BiancaWilkson) [inline code/modules/reagents/chemistry/reagents/medicine.dm:326]: You might want to add a column to data, so it doesn't fully override the other data, like blood types.

Comment (chuga-git) [reply code/modules/reagents/chemistry/reagents/medicine.dm:326]: agreed

**Review by BiancaWilkson — APPROVED** [review]
Why did data = 13? Code looks good.

**Comment by tipoima** [comment]
Synthflesh is used in tator medipen for its ingestion healing (and is a pretty big contributor of it).

You should either leave synthflesh's ingestion healing, or tweak the medipen so it's not suddenly underpowered.

**Review by S34NW — APPROVED** [review]

**Pull request merged**